roles/contiv_network/templates/netplugin.j2

```diff
@@ -1 +1 @@
-NETPLUGIN_ARGS='-plugin-mode {{netplugin_mode}} -vlan-if {{netplugin_if}} -vtep-ip {{node_addr}}'
+NETPLUGIN_ARGS='-plugin-mode {{netplugin_mode}} -vlan-if {{netplugin_if}} -vtep-ip {{node_addr}} -fwd-mode {{fwd_mode}}'
```

Comment:
ok, shall we wait for new netplugin release to be out before we merge this?

Also is there a good default value of `fwd_mode`? If so then we shall add that and document accepted values in `https://github.com/contiv/ansible/blob/master/roles/contiv_network/defaults/main.yml`